Assets/Scripts/Managers/Doors/DoorController.cs

```diff
@@ -62,7 +62,7 @@ public class DoorController : MonoBehaviour {
 				GameManager.Instance.LoadScene(
 					TransportTo switch {
 						TransportDestination.House => SceneDefs.HouseInteriorLevel,
-						TransportDestination.Special => SceneDefs.BossArenaLevel,
+						TransportDestination.Special => SceneDefs.ChurchCutscene,
```

Comment:
Works for now but I don't like this. Could've added a new TransportDestination (e.g. "Cutscene")